### ISSUE TITLE
Laptop for Simon

### ISSUE THREAD

**Issue by nelsonic** [dwyl/hq#565]
# Context

@SimonLab's laptop - that he uses for work-work every day - is falling apart; literally! 💻   💀  💔 
As the longest-serving and most valued member of our team, we want to buy him a **`new`** one. 

# _Framework_ ?

He has mentioned liking the Framework laptop:
https://frame.work/products/laptop-diy-12-gen-intel/configuration/edit
![image](https://user-images.githubusercontent.com/194400/170455977-5b4fa2cf-ea5f-4763-9e7b-2cf6d5950a68.png)

![image](https://user-images.githubusercontent.com/194400/170456023-79e99e8e-b44a-4ea3-b301-7c3391efcb2b.png)

Maxing out the config comes to **`$2,048`**. 

There is only one keyboard option available right now: **`US English`**
![image](https://user-images.githubusercontent.com/194400/170462134-d0baa7be-d8e9-44af-874f-6c4b9a30426f.png)

And the **Power Adapter** is  **US/Canada** ... could be worth researching a compatible UK/EU version.

If I were configuring this for myself these are the **Expansion Cards** I would chose:
+ 2 x USB-A
+ 1 x USB-C
+ 1 x HDMI 
![image](https://user-images.githubusercontent.com/194400/170462813-e038ea3f-fabb-4144-8734-0f1886402e51.png)

# Todo by @SimonLab 

Please either:

+ [x] _Confirm_ this is the laptop you want 

***or*** If you prefer a different laptop
+ [ ] please share links to alternatives. 

With shipping and import taxes it will probably come to **`$2,500`** (**`£2,000`**). So please work with that budget in mind.
(_please don't be "frugal", this is meant to last you at least **3 years**, it's on us and it's yours to **keep**_)

Keen to get this ordered and sent to you ASAP so you can use it!

**Comment by nelsonic** [comment]
@SimonLab please let us know if you want to order the (maxed out) Framework Laptop 🛠️ 
or _anything_ else that meets your needs. 💻 
Keen to get this done **ASAP**. 🙏

**Comment by SimonLab** [comment]
I like the idea of the Framework laptop as it is easy to upgrade or fix on  along term (hopefully the company will still be there in 3 years to provide the upgraded parts).

I think you saw the "us" version of the site as I can see the British version of the keyboard on the configuration edit page:
https://frame.work/gb/en/products/laptop-diy-12-gen-intel/configuration/edit
![image](https://user-images.githubusercontent.com/6057298/173324119-74c7a4cf-ab5a-4c50-8a6a-f24944d12dbf.png) 

The maxed out version is however more pricey on the UK site, but I don't think I would need 64Gb of memory or large ssd, so the price could go down to th £2,000 I think. I could still upgrade later on the parts if necessary!

**Comment by nelsonic** [comment]
@SimonLab good call. 
Please just confirm which 

+ [x] **keyboard language** you prefer
+ [x] **expansion cards** you want/need 
and if you want any spares e.g. stuff you might not use very often like (a spare) USB-C you could have in your laptop bag but only connect when needed and just have 3 USB-A and 1 HDMI permanently plugged in. 

It's cheaper to buy the RAM and SSD separately:
https://www.amazon.co.uk/Crucial-CT16G4SFRA32A-PC4-25600-SODIMM-260-Pin/dp/B07ZLCVKPV
<img width="1428" alt="image" src="https://user-images.githubusercontent.com/194400/173360093-52fd11a9-ede0-4de0-a020-b2458dd67f02.png">

https://www.amazon.co.uk/WD_BLACK-Internal-Gaming-Technology-Heatsink/dp/B08PMLYV9H
<img width="1507" alt="image" src="https://user-images.githubusercontent.com/194400/173360360-5a4b88c5-e7bd-4dcd-83e5-e6d54200691c.png">

You might as well Max both of these so you never have to think about running out of memory or storage.

**Comment by nelsonic** [comment]
Obvs if you prefer to order any of the components from somewhere _other_ than Amazon, please just share links! 👍

**Comment by SimonLab** [comment]
- Keyboard: British English (I'll get a french when we move back there!)
- expansion cars: 2 USB A; 1 USB C (for charging); 1 HDMI. I don't think I need any spare one.
- I don't mind ordering on Amazon if it's cheaper, I just need to make sure the ssd and memories are compatible, but I think they are:
> For memory, the Framework Laptop has two SO-DIMM sockets supporting DDR4 DRAM at up to DDR4-3200 speeds 
https://frame.work/gb/en/blog/storage-memory-and-wifi

Thanks again!!

**Comment by nelsonic** [comment]
@SimonLab thanks for your reply. Going to check with @iteles if we have your Bristol address on file (I think we do...) 
Will get this ordered for the August batch with your address for delivery. 

Just checked: **`BS5...`** ? 💭

**Comment by SimonLab** [comment]
> Just checked: BS5... ? 

Yes that's correct

**Comment by nelsonic** [comment]
cool. I will get this ordered as soon as I'm back at my desk. thanks. 👌

**Comment by nelsonic** [comment]
Pre-order maid and deposit paid:
![simon-framework-laptop-preorder-made](https://user-images.githubusercontent.com/194400/174341428-6df3a7ec-1419-482e-b5c7-0acb215104fc.png)

Will keep you informed of the delivery ETA. 👍

**Comment by nelsonic** [comment]
@SimonLab meanwhile ... your _current_ laptop is slowly dying ... 💻 💀 
And you _need_ something decent to get your work done. 💭 

So while we're patiently waiting for the Framework to arrive ... ⏳ 
I've sent you a **Lenovo X1 Carbon** https://www.lenovo.com/gb/en/d/thinkpad-x1-carbon [should arrive next Tuesday]
Because I know it has _excellent_ driver support for Ubuntu/Linux. :shipit: 
It's a refurbished [near-mint] unit. Don't worry, I didn't burn £2k on it! 😜 
it's got an i7, 16GB RAM, an ultra-fast SSD, _superb_ keyboard, decent WebCam, 14" 1080p Screen, USB-C charging etc. 
It has Windows 11 Pro installed but you can just ignore that, partition the SSD and dual-boot with your fave OS.

Once your Framework arrives, you can just transfer your work to the Framework (which we all agree is _way_ cooler and has the latest intel chip so will be faster) and you can either bring the **`X1`** to **PT** (when you visit) so we can have it as our new **Windows** "**Test**" **machine** [the old Win test laptop **`X230`** is being used to control the Laser cutter ...] 
***or*** if **`Jo`** likes it, she can totes keep it as her portable PhD Thesis writing machine. 😉

**Comment by SimonLab** [comment]
Thanks, I'll add Ubuntu to it, it should normally work without any issues.
I'll bring it back to you next time we'll be in Pt!!

**Comment by nelsonic** [comment]
Sweet @SimonLab thanks. 👍

**Comment by nelsonic** [comment]
@SimonLab FYI: "_Your order will be shipped within the next few business days._"

<img width="712" alt="image" src="https://user-images.githubusercontent.com/194400/188872180-c6dc1350-ea22-475c-aff2-f5e987ff433f.png">

We will place the order for the remaining components soon so you can start _using_ the laptop as soon as it arrives.

**Comment by SimonLab** [comment]
The Framework arrived at home this morning!
Thanks for checking the shipping progress

**Comment by nelsonic** [comment]
@SimonLab fantastic news! 🎉 
Let's get the SSD & RAM ordered _today_.
Do you have any preference for the online retailer? 

Looks like eBuyer (very reputable retailer we have bought from in the past) is _cheaper_ than Amazon!

<img width="1617" alt="image" src="https://user-images.githubusercontent.com/194400/190139610-b58f4bef-693e-4d9a-9ace-8572bc45dadf.png">

Please confirm the two products:
SSD: https://www.ebuyer.com/1127712-wd-black-sn850-2tb-m-2-pcie-4-0-nvme-ssd-solid-state-drive-wds200t1x0e
<img width="1381" alt="image" src="https://user-images.githubusercontent.com/194400/190139823-0e98a516-66fa-4193-abe3-e37388090a27.png">

RAM: https://www.ebuyer.com/1327274-exdisplay-crucial-32-gb-ddr4-3200-mhz-sodimm-260-pin-1-2-v-cl22-ebr1-ct32g4sfd832a (2 x 32Gb = 64Gb)
<img width="1381" alt="image" src="https://user-images.githubusercontent.com/194400/190141050-0822ef65-3986-4a28-a85e-bdf8b182b3b6.png">

Please confirm these are correct. Thanks.

**Comment by nelsonic** [comment]
Order placed with Next Day Delivery. ✅ 

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/194400/190162450-86196c77-5b3b-4af6-84a8-38784e7cfa6d.png">

**Comment by SimonLab** [comment]
The memories have arrived last week, and I was doing the laptop setup this morning.
Unfortunately after testing each ram while booting the laptop I concluded that one of the 32gb memory is not working.
I can still use the laptop with the one remaining memory and I'll continue the setup by installing ubuntu today
@nelsonic let me know what's the next step to replace and for you to get a refung

**Comment by nelsonic** [comment]
@SimonLab glad you got the Framework Laptop setup! 🎉 
Super lame that the RAM chip doesn't work. 🤦‍♂️ 
Obvs for 99% of work you're doing, you won't notice the lack of RAM (given that you have **`32GB`**...)

Just spent **`11+ minutes`** on the phone to ebuyer customer support: 

![image](https://user-images.githubusercontent.com/194400/192264211-0335841a-164b-40f3-a2f4-cdd312ddcac6.png)

most of the time was waiting on hold hearing the same BS message over-and-over ...
and then 1 minute to request the `RMA` (Return Merchandise Authorization).
Wish I had opened a _separate_ issue just for the RAM (Random Access Memory) ... 
so I could share it with the MD of ebuyer as a reference for what _not_ to do. 🤦‍♂️ 

![image](https://user-images.githubusercontent.com/194400/192269419-f398be69-e92d-4ca2-b6e0-c33cee22ca8b.png)

@SimonLab what day of the week are you most likely to be home for a collection? 💭 
![image](https://user-images.githubusercontent.com/194400/192269499-be0c76f7-67ac-4c73-8ef6-44734ae5d515.png)

***OR*** do you have access to a printer to print return labels 
![image](https://user-images.githubusercontent.com/194400/192269727-7e0ed656-3c01-4706-b556-b5b13f95093a.png)



RMA Number | 1633876
-- | --
RMA Reference | V-1633876-6743751
Created On | 26/09/2022
Status | Accepted

**Comment by nelsonic** [comment]
Collection arranged for **Wednesday 28<sup>th</sup> Sep** between 08:00 and 21:00 🙄 
Apologies for the absurd collection window. They have your phone number to call you when nearby. 🤞

**Comment by SimonLab** [comment]
The Parcel has just been collected.
9pm :grimacing: I'm sleeping already at this time

**Comment by nelsonic** [comment]
Glad it was collected. The replacement RAM chip should arrive tomorrow. 🤞

**Comment by nelsonic** [comment]
But can you run **`XCode`** on it? 😜

**Comment by nelsonic** [comment]
@SimonLab how's the Framework been these last couple of months? 💻 💭  **`#Feedback`** 🙏

**Comment by SimonLab** [comment]
So far it's good. There are a couple issues but it might be more linked to Ubuntu than the laptop itself.
I need to install all the tools for Flutter to see how responsive it will be.

**Comment by nelsonic** [comment]
Very curious to see if you can run Flutter. 🤞🏼
No reason it shouldn’t work. There are plenty of Android devs who use Ubuntu. 🦄

**Comment by nelsonic** [comment]
@SimonLab you've had the `Framework` laptop for just over 9 months. How has it been? 💭

**Comment by nelsonic** [comment]
@SimonLab hope you & fam are well 🎄 ❤️ 
and your `Framework` is still running smoothly. 💻 👌 

Closing as no further action to be taken on this issue. ✅
